jenkins/ompi/Dockerfile.centos7

```diff
@@ -0,0 +1,46 @@
+ARG OMPI_CI_OS_NAME=centos
+ARG OMPI_CI_OS_VERSION=7
```

Comment:
What is CentOS version for this tag? `7.6` or `7.7`? We need an exact version cause we use an exact HPCX build (e.g. `rhel7.6`).

Reply:
Changed to centos7.6.1810